bin/handler-slack-multichannel.rb

```diff
@@ -256,31 +263,44 @@ def post_data(notice, channel)
     http.use_ssl = true
 
     begin
-      req = Net::HTTP::Post.new("#{uri.path}?#{uri.query}")
-      if payload_template.nil?
-        text = slack_surround ? slack_surround + notice + slack_surround : notice
-        req.body = payload(text, channel).to_json
+      # Implement a retry strategy, with 5 tries max and a timeout of 10 seconds each
+      tries ||= 5
+      Timeout.timeout(10) do
+        begin
+          req = Net::HTTP::Post.new("#{uri.path}?#{uri.query}")
+          if payload_template.nil?
+            text = slack_surround ? slack_surround + notice + slack_surround : notice
+            req.body = payload(text, channel).to_json
+          else
+            req.body = notice
+          end
+          puts "request: #{req}"
```

Comment:
won't this contain auth? Can we add this behind a debug flag?

Reply:
It probably will. I honestly see no need at all to puts the whole request here, it was another case of change of indentation only, so I'd completely remove it tbh. Any complains to that idea?
It looks to me that the purpose of puts'ing the request here was for debugging purposes in case of an error raised so that the log shows somehow which alerts we are processing. This is now done in the exit messages on the rescue blocks, so it would be duplicate anyway

Reply:
Removed on https://github.com/sensu-plugins/sensu-plugins-slack/pull/82/commits/b0963fb2da6f467e250fd51738ea073286987ab8 so marking this as resolved.